### ISSUE TITLE
Links directing away from page

### ISSUE THREAD

**Issue by wookoouk** [inkline/inkline.io#44]
Can we please (happy to do it myself) change links in the docs to `href="#"` or similar, I am sick of clicking on something thinking it will toggle focus or become highlighted(active) only to redirect me to the front page of the site.

example: https://inkline.io/docs/components/list-group
The "link" and "router link" items should not direct the *current page* to `https://inkline.io/`

**Comment by alexgrozav** [comment]
Good point. It's a bit tricky because redirecting them to the current page will also cause them to have an active state. I think they should do nothing instead.

**Comment by wookoouk** [comment]
I would be happy just removing the hrefs but don't want to set off any
linting errors for people.

**Comment by wookoouk** [comment]
would you be ok with `href="#"` ?

**Comment by alexgrozav** [comment]
It would cause a page jump. Perhaps `href="javascript:void(0)"` could work.

**Comment by wookoouk** [comment]
That would probably be more cross-compatible than `<a href="#" onclick="return false;">`
I have forked the docs and will update all example links.
